lib/pay/lemon_squeezy/charge.rb

```diff
@@ -0,0 +1,50 @@
+module Pay
+  module LemonSqueezy
+    class Charge
+      attr_reader :pay_charge
+
+      delegate :processor_id, :customer, to: :pay_charge
+
+      def initialize(pay_charge)
+        @pay_charge = pay_charge
+      end
+
+      def self.sync(charge_id, object: nil, try: 0, retries: 1)
+        # Skip loading the latest subscription invoice details from the API if we already have it
+        object ||= ::LemonSqueezy::SubscriptionInvoice.retrieve(id: charge_id)
```

Comment:
what if we don't find any invoice and got 404 error? i suppose this case should be handled

I bumped into an `Error 404: No results were found for your request. '[{"status"=>"404", "title"=>"Not Found"}]' (LemonSqueezy::Error)` while accidentally sending test webhook to production server